static/js/recent_topics.js

```diff
@@ -402,7 +402,8 @@ function show_selected_filters() {
         for (const filter of filters) {
             $("#recent_topics_filter_buttons")
                 .find(`[data-filter="${CSS.escape(filter)}"]`)
-                .addClass("btn-recent-selected");
+                .addClass("btn-recent-selected")
+                .attr("aria-checked", "true");
```

Comment:
Do we need to toggle this field off in an else clause?  I think otherwise, `aria-checked` will remain true for anything that's every been checked.

Reply:
Since we rebuild filters everytime there is a change in filters. I noticed we didn't add aria for "All" filter, added it here.

Reply:
Shouldn't this set `aria-checked to false, not true, in the clear (unchecked) code path?

Reply:
Oh, I see, we remove the old copy from the DOM, and this logic is basically a workaround for the template that does that rebuild not supporting being passed the state in a way it can easily read.